source/style-guide/style/numbers.txt

```diff
@@ -83,13 +86,14 @@ parenthetically after the numeral.
      - Don't use
 
    * - two panels
-
-       zero probability
-
-       Enter **0** (zero). *(acceptable)*
      - two (2) panels
 
-       zero (0) probability
+   * - zero probability
+     - | zero (0) probability
+       | Enter **0** (zero). *(acceptable)*
+
+   * - Numbers with units
+     - 10MB
```

Comment:
My understanding of this table is that the "Use" and "Don't Use" columns are meant to show two variations on the same example. The "Use" column should be the accepted version of the copy, not a description.

Also, the lead-in to this table talks about numerals in parentheses, which wouldn't apply to this example.

I don't think this really belongs in this table.

Reply:
Done